arangod/Cluster/ClusterInfo.cpp

```diff
@@ -557,6 +558,93 @@ void ClusterInfo::loadClusterId() {
   }
 }
 
+/// @brief create a new collecion object from the data, using the cache if possible
+std::pair<std::shared_ptr<LogicalCollection>, uint64_t> ClusterInfo::buildCollection(
```

Comment:
It seems this could return a `CollectionWithHash` object. But this only "prettyness".

Reply:
Ok, will change it.

Reply:
done